### ISSUE TITLE
Server Unable To Boot Up

### ISSUE THREAD

**Issue by Ahren07** [sciguyryan/TheCrackPack#903]
Im having some issues with my server not booting up, ending with "Errno 32: Broken Pipe" Error log is posted below.

http://pastebin.com/ZCvqyQUm

**Comment by evknucklehead** [comment]
From my understanding, the "Errno 32: Broken Pipe" error is an indicator that the LaunchServer.sh script has ended unexpectedly because something in the script errored out. It doesn't specify why the server crashed, just that something went wrong while running the script.

Try setting the -Xms flag to 256M, instead of matching the -Xmx flag. I know for a lot of Java applications, matching the flags is useful, but with a program as big as Minecraft, it's better to start small and grow as needed. Also, try setting -Xmx to 3 or 4G, if you have it available, to give the server more room to grow.

You should also check the permissions for the server's directory and files. It's possible that something is choking while not being able to access or write to the level files.

The fact that only the memory flags are listed strikes me as odd as well, as there are some useful flags set by the script as it usually is. Granted, a couple of the flags are deprecated or removed in Java 8, but the rest of the flags still have some value.

**Comment by Ahren07** [comment]
I set those, but it still didnt work.

**Comment by Ahren07** [comment]
Still getting the Errno 32: Broken Pipe error

**Comment by evknucklehead** [comment]
After looking at the error report again, and comparing my local copy of LaunchServer.sh, it looks like the extra flags I was thinking of aren't used on the .sh script after all. Further, it looks like I was barking up the wrong tree on what was going on. (Call it road lag or something, been on a road trip with my grandfather starting earlier this week.)

It looks like something didn't load right with the wireless redstone mod. Reinstalling the server may fix the problem, but be sure to delete any WR-CBE files in the launcher's download folder first, so it can grab clean copies of the files.

**Comment by Ahren07** [comment]
So i will in fact lose my current world?

**Comment by evknucklehead** [comment]
No, you won't lose your world if you copy the world files to the new server folder.
